app/js/controllers/foyer/pensionsAlimentaires.js

```diff
@@ -24,13 +24,13 @@ angular.module('ddsApp').controller('FoyerPensionsAlimentairesCtrl', function($s
             })
             .value();
 
-        return result || 0;
+        return Math.round(result) || 0;
```

Comment:
Can't this be frustrating for the user if the value entered had decimals and they are systematically removed?

(sorry, I can't figure out where the data is coming from)